pay-api/src/pay_api/services/payment_account.py

```diff
@@ -151,7 +151,8 @@ def auth_account_id(self):
     def auth_account_id(self, value: str):
         """Set the auth_account_id."""
         self._auth_account_id = value
-        self._dao.auth_account_id = value
+        if self._dao.auth_account_id != value:
```

Comment:
If I didn't have these in here, this field would change, thus a version row would be created. 

It's an unfortunate problem having these services wrap DAO's.

Reply:
eft_short_names model is also versioned.

I assume we probably need to do this for that service as well?

Reply:
Probably a good idea lol, I'll add it in there.. honestly might be better to just gun the entire thing - but I'm trying to get to GCP with the least amount of changes